templating/render_without_controller.rst

```diff
@@ -22,7 +22,7 @@ can do this without creating a controller:
         # config/routes.yaml
         acme_privacy:
             path:         /privacy
-            controller:   Symfony\Bundle\FrameworkBundle\Controller\TemplateController
+            controller:   Symfony\Bundle\FrameworkBundle\Controller\TemplateController::templateAction
```

Comment:
That should work without the action method, because it's invokable `__invoke()` which in turn calls to `templateAction`.